### PR TITLE
fix(step): move classes to component-classes

### PR DESCRIPTION
**Pull request by CarolineGard**
move classes to component classes + cleanup of unused classes

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -86,18 +86,13 @@ export const step = {
   stepHorizontal: 'group/steph grid-rows-[auto_20px] grid-cols-[1fr_20px_1fr] flex-1 grid gap-y-16 items-center',
 
   stepDot: 'rounded-full border-2 h-20 w-20 transition-colors duration-300 i-text-$color-stepindicator-handle-icon',
-  stepDotVertical: '',
-  stepDotVerticalLeft: '',
   stepDotVerticalRight: 'col-start-2',
   stepDotHorizontal: 'row-start-2 justify-self-end',
-
   stepDotActive: 'i-border-$color-stepindicator-handle-border-active i-bg-$color-stepindicator-handle-background-active',
-  stepDotComplete: 'i-border-$color-stepindicator-handle-border-active i-bg-$color-stepindicator-handle-background-active',
```
Comment (BalbinaK) [inline index.js:95]: I don't think this should be removed 🤔 What was the reason for this change?

**Review by BalbinaK — COMMENTED** [review]
I saw there are also some [f-step](https://github.com/warp-ds/react/blob/alpha/packages/steps/src/step.tsx#L34) and ['step-line-h-l' ](https://github.com/warp-ds/react/blob/alpha/packages/steps/src/step.tsx#L85)classes in step.tsx. Should those be looked at, too?

File: index.js (hunk 1 above)
Comment (CarolineGard) [reply index.js:95]: It was a duplicate of the `stepDotActive`, as they contain the same styling

**Comment by CarolineGard** [comment]
> I saw there are also some [f-step](https://github.com/warp-ds/react/blob/alpha/packages/steps/src/step.tsx#L34) and ['step-line-h-l' ](https://github.com/warp-ds/react/blob/alpha/packages/steps/src/step.tsx#L85)classes in step.tsx. Should those be looked at, too?

The current classes here e.g. `grid-cols-[1fr_20px]` replaces `f-step` and `step-line-h-l`. So I've updated the components as well, but I'm waiting for these changes first before I can add it there:)

File: index.js (hunk 1 above)
Comment (BalbinaK) [reply index.js:95]: Ah, sorry then 🙃

**Review by BalbinaK — APPROVED** [review]

**Pull request merged**